pkg/ingester/ingester.go

```diff
@@ -1928,6 +1927,54 @@ func (i *Ingester) openExistingTSDB(ctx context.Context) error {
 	return nil
 }
 
+// getAllUsersWithTSDB finds all users with a TSDB on the filesystem.
+func (i *Ingester) getAllUsersWithTSDB() (map[string]struct{}, error) {
```

Comment:
This name is fine, but we could use `getAllTSDBUserIDs` just to show that we'll be getting UserIDs back to act on?